pkg/summarizer/flakiness_test.go

```diff
@@ -0,0 +1,460 @@
+package summarizer
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/testgrid/pb/state"
+)
+
+func TestIsWithinTimeFrame(t *testing.T) {
+	cases := []struct {
+		name      string
+		column    *state.Column
+		startTime int
+		endTime   int
+		expected  bool
+	}{
+		{
+			name: "column within time frame returns true",
+			column: &state.Column{
+				Started: 1.0,
+			},
+			startTime: 0,
+			endTime:   2,
+			expected:  true,
+		},
+		{
+			name: "column outside of time frame returns false",
```

Comment:
Maybe test for other bound Started:1.0.    startTime:3. endTime:5

Reply:
Done